grafana-11.2.yaml

```diff
@@ -61,7 +61,7 @@ pipeline:
       cp conf/sample.ini ${{targets.destdir}}/etc/grafana/grafana.ini.example
 
 subpackages:
-  - name: grafana-oci-compat
+  - name: ${{package.name}}-oci-compat
```

Comment:
This is missing a `provides` and so now image builds are picking up old versions of this package.